src/main/java/org/mule/extension/ftp/api/ftp/FtpFileAttributes.java

```diff
@@ -94,10 +105,6 @@ public void setSize(long size) {
     this.size = size;
   }
 
-  /**
-   * {@inheritDoc}
-   */
```

Comment:
Provide the comments in all the methods.

Reply:
added